docs/en/observability/apm/collect-application-data/index.asciidoc

```diff
@@ -0,0 +1,101 @@
+[[apm-collect-application-data]]
+= Collect application data
+
+[discrete]
+== Language-specific options
+
+Use Elastic APM agents or an OpenTelemetry language SDK to instrument a service in the language its written in:
+
+* <<apm-agents,*Elastic APM agents*>>: Elastic APM agents are instrumentation libraries written in the same language as your service.
+* <<apm-open-telemetry,*OpenTelemetry*>>: OpenTelemetry is an open source set of APIs, SDKs, tooling, and integrations that enable the capture and management of telemetry data from your services and applications.
+** This option includes Elastic Distributions of OpenTelemetry, which are customized versions of https://opentelemetry.io/docs/languages/[OpenTelemetry language SDKs] that are optimized to work with an Elastic backend.
+
+*Not sure which method is right for you?* Compare the available options below.
+
+[discrete]
+=== Capabilities
+
+////
+TO DO:
+Is this table even helpful?
+Is it missing anything that would make it more helpful?
+////
```

Comment:
cc @mlunadia / @AlexanderWert / @alex-fedotyev 

Here's the built page: https://observability-docs_bk_4230.docs-preview.app.elstc.co/guide/en/observability/master/apm-collect-application-data.html#_capabilities

Reply:
The open questions are:

1. Is this table even helpful?
2. Is it missing anything that would make it more helpful?
3. Are we ok with maintaining this table?
4. Are we ok with the gaps it exposes?

Reply:
@bmorelli25 do you have any thoughts on these questions before we merge this PR?

Reply:
LGTM. Let's ship it.